zerver/actions/message_edit.py

```diff
@@ -876,6 +876,20 @@ def exclude_duplicates_from_subscription(
 
             return subs
 
+        # We also need to include users who are not subscribed to new stream
+        # but are subscribed to the old stream.
+        old_stream_subscriptions = get_active_subscriptions_for_stream_id(
+            stream_being_edited.id, include_deactivated_users=True
```

Comment:
Will these deactivated users end up being included in the users to be notified?

Reply:
Deactivated users can't receive events, so I think this should just be `False`?

Reply:
Okay, that makes sense. Changed to `False`.